core/admin/Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM alpine:3.8
+FROM alpine:3.10
 # python3 shared with most images
 RUN apk add --no-cache \
     python3 py3-pip git bash \
   && pip3 install --upgrade pip
-RUN pip3 install git+https://github.com/usrpro/MailuStart.git#egg=mailustart
+# Shared layer between admin, rspamd, postfix, dovecot, unbound and nginx
+RUN pip3 install git+https://github.com/Mailu/MailuStart.git#egg=mailustart
```

Comment:
Is this really needed? https://github.com/Mailu/Mailu/blob/master/core/admin/start.py does not use any (shared) functionality?

Reply:
You are right.
And if this layer gets removed we should also change the comment in the other dockerfiles :)

(Maybe we should create mailu base-images so we could do
`FROM mailubase+python+mailustart`
or something, instead of the same text again and again in so many files…)

Reply:
Creating a base image is certainly an approach. I would like to have time and provide Socrate properly before then, so we can properly pull it from pypi.

Reply:
IMO what @kaiyou says makes sense, if going the baseimage approach this would either be a separate effort/PR, or a good chance to combine with his socrate efforts, which seem great to me …

Reply:
… and all of us didn’t notice admin uses `MailuStart` in 
https://github.com/Mailu/Mailu/blob/master/core/admin/mailu/configuration.py#L2
:joy: :joy: :joy:

Reply:
But with #1048 @kaiyou removed `MailuStart`. So I would prefer to merge it first and then rebase this PR here. 
@Nebukadneza maybe you can look into #1048, wheter the issues you had are now resolved and approve the PR :+1:

Reply:
ah, and then rebase there … that makes sense. Will do!